R/constants.R

```diff
@@ -116,6 +116,10 @@ aglu.BIO_GRASS_YIELD_KGCM2 <- 0.34
 aglu.BIO_TREE_FERT_IO_GNM2 <- 3.36
 aglu.BIO_TREE_YIELD_KGCM2 <- 0.345
 
+# Water characteristics for biomass (from Vaibhav's paper)
```

Comment:
DOI 10.1007/s11027-013-9497-4
Chaturvedi et al. 2015, Climate mitigation policy implications for global irrigation water demand, Mitig Adapt Strateg Glob Change (2015) 20:389-407.